housekeeper/store/api/find.py

```diff
@@ -40,6 +43,10 @@ def _get_version_bundle_query(self) -> Query:
         """Return version bundle query."""
         return self.Version.query.join(Version.bundle)
 
+    def _get_file_tag_query(self) -> Query:
+        """Return file tag query."""
+        return self.File.query.join(File.tags)
+
     def bundle(self, name: str = None, bundle_id: int = None) -> Bundle:
```

Comment:
rename the function to ```get_bundle_by_id ````
Make sure the id name is explicit for the id it is accessing. i.e. internal_id, entry_id etc.

Reply:
The function also allows for retrieving bundles by name, so it does not make sense to name it `get_bundle_by_id`. But I will split it into two smaller functions `get_bundle_by_name` and `get_bundle_by_id`.

Reply:
Fixed